pkg/function/kubeops_test.go

```diff
@@ -195,54 +198,71 @@ func (s *KubeOpsSuite) TestKubeOps(c *C) {
 	defer cancel()
 	tp := param.TemplateParams{}
 	action := "test"
+	serviceName := fmt.Sprintf("%s-%s", testServiceName, rand.String(8))
 	type resourceRef struct {
 		gvr       schema.GroupVersionResource
-		name      string
 		namespace string
 	}
 	for _, tc := range []struct {
+		name        string
 		bp          crv1alpha1.Blueprint
 		expResource resourceRef
+		errChecker  Checker
 	}{
 		{
-			bp: newCreateResourceBlueprint(createInSpecsNsPhase(s.namespace)),
+			// Do not expect the resource to be created with the provided name
+			name: fmt.Sprintf("%s-%s", testServiceName, rand.String(8)),
```

Comment:
which name is this? is this the same name that we are creating the resource with?